REST/PowerShell/Reports/DeploymentsByMachineReport.ps1

```diff
@@ -0,0 +1,263 @@
+$ErrorActionPreference = "Stop";
```

Comment:
This will need to be updated to support TLS 1.2 and 1.3: https://github.com/OctopusDeploy/OctopusDeploy-Api/blob/master/REST/PowerShell/Administration/GetUsageCounts.ps1#L328C1-L330C8